### DISCUSSION TITLE
UTMStack update fails and breaks UTMStack

### DISCUSSION THREAD

**Discussion by pwhitney5524** [utmstack/UTMStack#883] [Q&A]
Getting an error when updating UTMStack 10.5.3 to the latest version using the following documentation: https://docs.utmstack.com/Upgrade/README.html.

We initially installed UTMStack using the image ISO about 6 months ago.

The error I am getting when running the latest installer is the following:

_Running post installation scripts. This may take a while.
Securing ports 9200, 5432 and 10000
utmstack_node1
overall progress: 0 out of 1 tasks
1/1: starting  [============================================>      ]
service update paused: update paused due to failure or early termination of task gan2kznrk80h7wv0c933m03bl
exit status 1_

After that, UTMStack simply stops working and I get the 502 error from NGINX.

Does anyone know how to fix this so the update may succeed?

**Comment by c3s4rfred** [comment]
Hi @pwhitney5524, did you try to run the installer again? Also check your network, with better bandwidth issues will be less. Check the hardware and software requirements, it's important.

Best regards

**Comment by pwhitney5524** [comment]
I retried and here is what I am getting now:

Installing reverse proxy [OK]
Waiting for Backend to be ready. This may take a while.
backend start failure: <nil>

This is running on a virtual machine using an all-flash SAN with plenty of resources assigned to the VM.

Ideas?

**Reply by c3s4rfred** [reply]
You are installing on normal VM or LXC ? LXC is not supported

**Reply by pwhitney5524** [reply]
We installed it as a Hyper-V VM using the ISO image and it installed just fine.

Updating to the latest version is the problem.
